Bottom: 08ce7e1c78b17e7a02cd553feb17cd5ffa5f2546
Top:    fba6c9c590cddc149edad0d6054c96229eaefc13
Author: Johannes Rudolph <johannes_rudolph@gmx.de>
Date:   2010-05-31 09:58:46 +0200

update to specs 1.6.5-SNAPSHOT


---

diff --git a/objectformatter/pom.xml b/objectformatter/pom.xml
index 2fe8f30..a07c026 100644
--- a/objectformatter/pom.xml
+++ b/objectformatter/pom.xml
@@ -14,6 +14,11 @@
       <name>Scala-Tools Maven2 Repository</name>
       <url>http://scala-tools.org/repo-releases</url>
     </repository>
+    <repository>
+      <id>scala-tools.org snapshots</id>
+      <name>Scala-Tools Maven2 Repository</name>
+      <url>http://scala-tools.org/repo-snapshots</url>
+    </repository>
   </repositories>
 
   <pluginRepositories>
@@ -42,9 +47,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.specs</groupId>
-      <artifactId>specs</artifactId>
-      <version>1.3.1</version>
+      <groupId>org.scala-tools.testing</groupId>
+      <artifactId>specs_2.8.0.RC3</artifactId>
+      <version>1.6.5-SNAPSHOT</version>
       <scope>test</scope>
     </dependency>
     <dependency>
diff --git a/objectformatter/src/test/scala/net/virtualvoid/string/ESParserSpecs.scala b/objectformatter/src/test/scala/net/virtualvoid/string/ESParserSpecs.scala
index 82f4675..09e80d9 100644
--- a/objectformatter/src/test/scala/net/virtualvoid/string/ESParserSpecs.scala
+++ b/objectformatter/src/test/scala/net/virtualvoid/string/ESParserSpecs.scala
@@ -64,7 +64,7 @@ object ParserSpecs extends Specification {
   def toks(inner:FormatElement*) = FormatElementList(List(inner:_*))
 
   import org.specs.specification.Example
-  def parseCorrectly(e: =>Example) = { currentSut.verb += " parse correctly"; e }
+  def parseCorrectly = addToSusVerb(" parse correctly")
 }
 
-class ParserSpecsTest extends runner.JUnit4(ParserSpecs)
\ No newline at end of file
+class ParserSpecsTest extends runner.JUnit4(ParserSpecs)
